expediagroup/sdk/generator/client/visitors/models.py

```diff
@@ -243,6 +262,7 @@ def get_models(parser: OpenAPIParser, model_path: Path) -> dict[str, object]:
     models: dict[str, DataModel] = parse_datamodels(parser)
     discriminators: list[Discriminator] = parse_discriminators(parser=parser, models=models)
 
+    set_other_responses_models([operation for operation in parser.operations.values()])
```

Comment:
Do we need to set error responses on operations? We get_error_models from it and add it to the return object